documentation-standards.md

```diff
@@ -18,7 +18,7 @@ Here's an example of the PHPdoc annotating the `wp cron event schedule` command:
  * : The hook name.
  *
  * [<next-run>]
- * : A Unix timestamp or an English textual datetime description compatible with `strtotime()`. Defaults to now.
+ * : A Unix timestamp or an English textual date time description compatible with `strtotime()`. Defaults to now.
```

Comment:
Here, `datetime` refers to a type of data (date + time in one single scalar value). This is a commonly used term in software engineering.

Please revert this particular change.